packages/canary-web-components/src/components/ic-tree-view/ic-tree-view.stories.mdx

```diff
@@ -723,3 +723,49 @@ An example where the tree item is rendered as a link.
   <ic-tree-item label="Hot chocolate" disabled="true" href="#"></ic-tree-item>
 </ic-tree-view>
 ```
+
+### Expanded
+
+An example with expanded prop set to true on a parent tree item.
```

Comment:
```suggestion
An example with the `expanded` prop set to `true` on a parent tree item.
```
Also in the React stories

Sorry, I realise I missed this when reviewing the focus inset PR!